projects/packages/forms/src/contact-form/class-feedback.php

```diff
@@ -1144,4 +1145,42 @@ private function get_computed_consent( $post_data, $form ) {
 
 		return false;
 	}
+
+	/**
+	 * Get a field by its original form ID.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param string $id Original form field ID.
+	 * @return Feedback_Field|null
+	 */
+	public function get_field_by_id( $id ) {
+		if ( ! is_string( $id ) || $id === '' ) {
+			return null;
+		}
+		foreach ( $this->fields as $field ) {
+			$form_field_id = $field->get_form_field_id();
+			if ( is_string( $form_field_id ) && $form_field_id !== '' && $form_field_id === $id ) {
+				return $field;
+			}
+		}
+		return null;
+	}
+
+	/**
+	 * Get a field render value by its original form ID.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param string $id Original form field ID.
+	 * @param string $context Render context.
+	 * @return string
+	 */
+	public function get_field_value_by_id( $id, $context = 'default' ) {
```

Comment:
should we call this. 
`get_field_value_by_form_field_id` ?

Reply:
Updated both method names!